README.md

```diff
@@ -54,22 +62,18 @@ The [HiveMind repository](https://github.com/SwarmUS/HiveMind) is included as a
 
 ## Building
 
-Assuming you have a working ROS + Gazebo setup on you system, you need only clone this repository in your Catkin workspace, initialise the git submodules, and build it. 
+Assuming you have a working ROS + Gazebo setup on you system, you need only clone this repository in your Catkin workspace, initialise the git submodules, and build it. You will also need to use [rosdep](http://wiki.ros.org/rosdep) to manage some external package dependencies. Refer to the following instruction for building:
 
 ```
 cd ~/catkin_ws/src
 git clone https://github.com/SwarmUS/SwarmUS-ROS.git
 cd SwarmUS-ROS
 git submodule update --init --recursive
-cd ~/catkin_ws && catkin_cmake
-```
-
-You might need to use [rosdep](http://wiki.ros.org/rosdep) to manage some external package dependencies. The following command installs the dependencies for all the packages of a given workspace :
-
-```
+sh scripts/install dependencies.sh
```

Comment:
Maybe just note that it's for ubuntu based systems

Reply:
True. I assumed all systems would be Ubuntu-based due to the inherent ROS dependency but Debian is also technically supported.